common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -407,13 +446,13 @@
             "off"
         ],
         "import/namespace": [
-            2
+            "off"
```

Comment:
import/namespace seems good to error on

Reply:
The typescript-eslint folks [say it's superfluous](https://typescript-eslint.io/docs/linting/troubleshooting#eslint-plugin-import) in TS:

> However there are a few rules which can cause your lints to be really slow, because they cause the plugin to do its own parsing, and file tracking. This double parsing adds up for large codebases.
>
> There are many rules that do single file static analysis, but we provide the following recommendations.
>
> We recommend you do not use the following rules, as TypeScript provides the same checks as part of standard type checking:
>
>```
>import/named
>import/namespace
>import/default
>import/no-named-as-default-member
>```
>
>The following rules do not have equivalent checks in TypeScript, so we recommend that you only run them at CI/push time, to lessen the local performance burden.
>
>```
>import/no-named-as-default
>import/no-cycle
>import/no-unused-modules
>import/no-deprecated
>```